data_model/models.py

```diff
@@ -0,0 +1,109 @@
+from django.contrib.postgres import fields as pg_fields
+from django.db import models
+
+
+class IETFReport(models.Model):
+    rrname = models.CharField(max_length=100)
+    rrtype = models.CharField(max_length=100)
+    rdata = pg_fields.ArrayField(models.CharField(max_length=100))
+    time_first = models.DateTimeField()
+    time_last = models.DateTimeField()
+
+
+class BaseDataModel(models.Model):
+    evaluation = models.CharField(
+        max_length=100, null=True
+    )  # classification/verdict/found/score/malscore
+    # HybridAnalysisObservable (verdict), BasicMaliciousDetector (malicious),
+    # GoogleSafeBrowsing (malicious), Crowdsec (classifications),
+    # GreyNoise (classification), Cymru (found), Cuckoo (malscore),
+    # Intezer (verdict/sub_verdict), Triage (analysis.score),
+    # HybridAnalysisFileAnalyzer (classification_tags)
+    external_references = pg_fields.ArrayField(
+        models.URLField(), null=True
+    )  # link/external_references/permalink/domains
+    # Crowdsec (link), UrlHaus (external_references), BoxJs,
+    # Cuckoo (result_url/permalink), Intezer (link/analysis_url),
+    # MalwareBazaarFileAnalyzer (permalink/file_information.value), MwDB (permalink),
+    # StringsInfo (data), Triage (permalink), UnpacMe (permalink), XlmMacroDeobfuscator,
+    # Yara (report.list_el.url/rule_url), Yaraify (link),
+    # HybridAnalysisFileAnalyzer (domains),
+    # VirusTotalV3FileAnalyzer (data.relationships.contacted_urls/contacted_domains)
+    related_threats = pg_fields.ArrayField(
+        models.CharField(max_length=100), null=True
+    )  # threats/related_threats
```

Comment:
I would add the `tags` field (like in the File) here too.
So that we could have generic tags, like `phishing` and the inside the related_threats we have pointers to other IOCs